wdio.grid.conf.js

```diff
@@ -0,0 +1,294 @@
+const drivers = {
+    chrome: { version: '88.0.4324.96' }, // https://chromedriver.chromium.org/
```

Comment:
Isn't it possible to have 2 files containing only different configuration setups and an additional file containing a common config?